torch/_tensor_docs.py

```diff
@@ -4102,6 +4102,75 @@ def callable(a, b) -> number
     >>> torch.equal(b, c)
     False
 
+
+view(dtype) -> Tensor
+
+Returns a new tensor with the same data as the :attr:`self` tensor but of a
+different :attr:`dtype`.
+
+The returned tensor shares the same data and must have the same number
```

Comment:
I suggest replacing this paragraph with 

":attr:\`dtype\` must have the same number of bytes per element as :attr:\`self\`'s dtype."

And attaching that as a second sentence to the first paragraph.

Reply:
fixed